src/concrete/ml/sklearn/base.py

```diff
@@ -1311,9 +1318,22 @@ def fit(self, X: Data, y: Target, **fit_parameters):
         assert self.sklearn_model is not None, self._sklearn_model_is_not_fitted_error_message()
 
         # Convert the tree inference with Numpy operators
+        enable_rounding = os.environ.get("USE_ROUNDING", "True") == "True"
+
+        if not enable_rounding:
+            warnings.simplefilter("always")
+            warnings.warn(
+                "Using Concrete Tree-Based models without the `rounding feature` is deprecated. "
```

Comment:
weird to capitalize `Tree-Based`, our doc always mentions "tree-based" so I would keep it that way !

Reply:
ok